integration_tests/pages/refer/newReferral.ts

```diff
@@ -0,0 +1,30 @@
+import helpers from '../../support/helpers'
+import Page from '../page'
+
+export default class NewReferralPage extends Page {
```

Comment:
As per my previous comment, should this be `FindPersonPage`? I suppose arguably it could be either, I think users would call this the "page for searching for a person" rather than a new referral page, but what do you think?